hoomd/write/gsd.py

```diff
@@ -90,43 +100,36 @@ class GSD(Writer):
         * constraints/
         * pairs/
 
-
-    .. seealso::
-
+    See Also:
         See the `GSD documentation <http://gsd.readthedocs.io/>`_ and `GitHub
         project <https://github.com/glotzerlab/gsd>`_ for more information on
```

Comment:
I recommend making these anonymous links with two underscores. Otherwise if you have another link somewhere in HOOMD with the text "GSD documentation" or "GitHub project," it will throw a warning.
```suggestion
        See the `GSD documentation <http://gsd.readthedocs.io/>`__ and `GitHub
        project <https://github.com/glotzerlab/gsd>`__ for more information on
```